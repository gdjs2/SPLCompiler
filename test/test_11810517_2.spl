int a;

int test (){
  int b = 0xFF;
  c = 5;
}