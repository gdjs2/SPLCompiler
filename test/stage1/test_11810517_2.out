Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      ExtDecList (1)
        VarDec (1)
          ID: a
      SEMI
    ExtDefList (3)
      ExtDef (3)
        Specifier (3)
          TYPE: int
        FunDec (3)
          ID: test
          LP
          RP
        CompSt (3)
          LC
          DefList (4)
            Def (4)
              Specifier (4)
                TYPE: int
              DecList (4)
                Dec (4)
                  VarDec (4)
                    ID: b
                  ASSIGN
                  Exp (4)
                    INT: 255
              SEMI
          StmtList (5)
            Stmt (5)
              Exp (5)
                Exp (5)
                  ID: c
                ASSIGN
                Exp (5)
                  INT: 5
              SEMI
          RC
